IO.Eventuate.Tram.Tests/docker-compose.yml

```diff
@@ -28,3 +38,31 @@ services:
       KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
       KAFKA_CONFLUENT_LICENSE_TOPIC_REPLICATION_FACTOR: 1
       CONFLUENT_SUPPORT_CUSTOMER_ID: 'anonymous'
+  cdcservice:
+    image: eventuateio/eventuate-cdc-service:${CDC_SERVICE_DOCKER_VERSION:?You must set the CDC_SERVICE_DOCKER_VERSION environment variable.}
+    depends_on:
+    - kafka
+    - mssql
+    environment:
+      EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS: kafka:29092
+      EVENTUATELOCAL_ZOOKEEPER_CONNECTION_STRING: zookeeper:2181
+      EVENTUATE_CDC_READER_READER1_TYPE: polling
+      EVENTUATE_CDC_READER_READER1_DATASOURCEURL: jdbc:sqlserver:LAPTOP-7G6A2VOR;databaseName=TramDb
+      EVENTUATE_CDC_READER_READER1_DATASOURCEUSERNAME: sa
```

Comment:
This hostname is incorrect. `LAPTOP-7G6A2VOR` should be `mssql` - the name of the MS SQL server container

You just need to copy https://github.com/eventuate-tram/eventuate-tram-core-dotnet/blob/master/IO.Eventuate.Tram.IntegrationTests/docker-compose.yml

Reply:
Done